src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -50,11 +49,11 @@ public static ReadOnlyAddressBook getSampleAddressBook() {
     }
 
     /**
-     * Returns a tag set containing the list of strings given.
+     * Returns a study group set containing the list of strings given.
      */
-    public static Set<Tag> getTagSet(String... strings) {
+    public static Set<StudyGroupTag> getStudyGroupSet(String... strings) {
```

Comment:
Just a nit: can consider if `getStudyGroupTagSet` would be a better method name, if not can ignore (since it's also quite long)